core/pom.xml

```diff
@@ -240,6 +240,11 @@ THE SOFTWARE.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>net.sf.kxml</groupId>
+        <artifactId>kxml2</artifactId>
+        <version>2.3.0</version>
```

Comment:
what where the reasons for not using the stock JDK SAX parser which is XML 1.1 compliant?

Reply:
I was having issues getting Xstream to use the correct parser if i recall correctly.  I've got a bit better understanding of how things work now, and will go back and see if that is still the case.

Reply:
@jtnord - after quite a bit of investigation, I was able to use the stock JDK sax parser, but this introduced a number of new issues.  
- The stock parser wants to write its own <?xml ...> header, and when using the HierarchicalStreamDriver (which is what XStream2 uses), you are not given the option to change it (and its defaulted to <?xml version='1.0'>
- Switching to use an XMLStreamWriter gives you the ability to set both the character encoding and xml version.  However, this then requires a fairly significant amount of change in our XStream implementation.

I think that for now the best choice is to use the KXml2 driver.  It is more performant than the Xpp3 driver, and if we see value in moving to a stock JDK implementation in the future, we can tackle that as a separate issue.

Side Note: The original implementation was a bit inconsistent in which drivers were being used.  I've cleaned that up to ensure that all of the XML operations are now using the same driver.